SwanContents/swancontents/filemanager/swanfilemanager.py

```diff
@@ -337,43 +343,6 @@ def update(self, model, path):
 
         return super(LargeFileManager, self).update(model, path)
 
-    def delete(self, path, force=False):
-        """Delete a file/directory and any associated checkpoints."""
-        path = path.strip('/')
-        if not path:
-            raise web.HTTPError(400, "Can't delete root")
-        self.delete_file(path, force)
-        self.checkpoints.delete_all_checkpoints(path)
```

Comment:
So now we won't delete the checkpoints?

Reply:
This was not the code we added. We added the possibility to delete a folder that contained some files inside (i.e the `.swanproject`). But with the force delete enabled, upstream now works the same way as we did (so, it still deletes checkpoints).